3.0.20/reference.json

```diff
@@ -1489,6 +1489,15 @@
                 "default-value": 0,
                 "default-meaning": "No placement will be done.",
                 "doc": "Controls vertical spacing between placements of placement type `grid` or `alternating-grid`."
+            },
+            "offset": {
+                "css": "shield-offset",
+                "type": "float",
+                "expression": true,
+                "default-value": 0.0,
+                "default-meaning": "Will not be offset.",
+                "doc": "Offsets a line or a polygon a number of pixels parallel to its actual path. Positive values move the line left, negative values move it right (relative to the directionality of the line).",
```

Comment:
> Offsets a line or a polygon

Aren't we talking about a shield instead?

Reply:
Good point. It offsets underlaying geometry, which causes texts or shields to move by the offset in case of `placement="line"`, but not in the case of `placement="grid"`. In such case it reduces or expands underlaying polygon area where individual placements can go. I will try to describe it better.